src/components/PrizePage/Prizes.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import { basename } from 'path';
 import { graphql, useStaticQuery } from 'gatsby';
-import Box from '@material-ui/core/Box';
+/* import Box from '@material-ui/core/Box';
```

Comment:
Delete the commented out code, it is generally better practice to not have all this code lying around that will potentially never get used again.